agent/wptdriver/wpt_test.cc

```diff
@@ -288,6 +290,8 @@ bool WptTest::Load(CString& test) {
         else if (!key.CompareNoCase(_T("continuousVideo")) &&
                  _ttoi(value.Trim()))
           _continuous_video = true;
+		else if (!key.CompareNoCase(_T("timeout")))

```

Comment:
between the braces and the indent, it's really hard to see where this clause is supposed to be.